apiabroad/payment-service
# [DAT-622] Payments Table

# [DAT-622](https://apiabroad.atlassian.net/browse/DAT-622)

Creates the base project structure to work on, containing:

- Models
- Schemas
- Tests
- pre-commit

based on fastapi

## Dependencies

- None yet

## Type of change

## How to test

1. Follow the [readme.md ](https://github.com/apiabroad/payment-service/blob/804130e1134ed14455d04c3611f31dbf4eb2838c/README.md)instructions to setup the payment service
2. Check that the table contains the columns / indexes described in [this ticket](https://apiabroad.atlassian.net/browse/DAT-622)
3. Additionally run `poetry run pytest` to run the unit tests
